packages/wallet-api-manifest-validator/src/validator.ts

```diff
@@ -0,0 +1,72 @@
+import Ajv, { ErrorObject } from "ajv";
+import ajvErrors from "ajv-errors";
+import { Logger } from "@ledgerhq/wallet-api-core";
+import SchemaJSON from "./schema/schema.json";
+
+export interface OptionsParams {
+  details?: boolean;
+  enableState?: boolean;
+  fileName?: string;
+}
+
+/**
+ *  Verify if your JSON meets the requirements for Ledger Live App manifest submission
+ *  @param {JSON} manifest - json file.
+ *  @returns {boolean}
+ *
+ *  @param {ValidateManifestParams=} options :
+ *      @param {boolean=} details - describe errors when they occur
+ *      @param {boolean=} enableState - Result description (e.g. show in console "The JSON file do not correspond to the schema")
+ *      @param {string=} fileName - file name, practical to validate multiple files via multiple calls
+ */
+export function validateManifest(
+  manifest: JSON,
+  options?: OptionsParams | undefined
+): boolean {
+  const { details, enableState, fileName } = {
+    details: false,
+    enableState: false,
+    fileName: "",
+    ...options,
+  };
+
+  const validate = ajvErrors(new Ajv({ allErrors: true })).compile(SchemaJSON);
```

Comment:
We should move this outside the function to avoid compiling on every call of `validateManifest`